_inc/client/plans/single-product-backup.jsx

```diff
@@ -2,18 +2,63 @@
  * External dependencies
  */
 import React from 'react';
+import PropTypes from 'prop-types';
 import { translate as __ } from 'i18n-calypso';
 
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ExternalLink from 'components/external-link';
+
 import './single-product-backup.scss';
 
-export function PlanRadioButton( {
-	checked,
-	currencySymbol,
-	onChange,
-	planName,
-	radioValue,
-	planPrice,
-} ) {
+export function PlanPriceDisplay( { backupPlanPrices, currencySymbol } ) {
+	const dailyBackupYearlyPrice = backupPlanPrices.jetpack_backup_daily.yearly;
```

Comment:
Do we need any sort of property existence guards here?

Reply:
In this case I don't think so. This is really just a helper component for the header component that comes in the final PR, where the property is guaranteed to exist. I prefer checking for existence much closer to the Redux connection so that helper components like this can be kept free of prop validation to keep them as simple as possible.

Reply:
> the property is guaranteed to exist

These are famous last words 😉